### DISCUSSION TITLE
fail to add new Vegetation Bending property group In Material Editor

### DISCUSSION THREAD

**Discussion by hequan-Bill** [o3de/o3de#15092] [Q&A]
I added new  Vegetation Bending property group based on vertex deformation tutorial,but in the Material Editor  these assets cannot be displayed
<img width="458" alt="捕获" src="https://user-images.githubusercontent.com/82749476/225017322-f7371bea-77c3-440f-816b-2b87e1fb3242.PNG">
<img width="283" alt="1" src="https://user-images.githubusercontent.com/82749476/225017447-11855561-899a-47c2-8681-d6eebfceb289.PNG">

I don't know how to solve it，I hope to get your help

**Comment by AMZN-alexpete** [comment]
@amzn-tommy might now more, but I think the tutorial may need to be updated to be compatible with changes that were made to the materials and shaders since the tutorial was made.

**Reply by amzn-tommy** [reply]
Yes, the tutorial itself is out of date. The shaders have gone through a significant re-factor since then, so anything related to how to modify StandardPbr to use a custom vertex shader is out of date. Ideally, at some point the tutorial is updated to demonstrate vertex deformation via material canvas.

Though by itself, adding a property group to a material type should not have changed much AFAIK. Are there any errors in the AssetProcessor, or in the console/MaterialEditor.log?
